GameDefinition/Content/ItemKindActor.hs

```diff
@@ -474,6 +474,24 @@ alligator = ItemKind
                , ("eye 6", COrgan), ("ear 9", COrgan)
                , ("animal brain", COrgan) ]
   }
+octopus = ItemKind
+  { isymbol  = 'o'
+  , iname    = "Giant octopus"
+  , ifreq    = [ ("animal", 100), ("mobile", 1), ("mobile animal", 100) ]
+  , iflavour = zipPlain [BrMagenta]
+  , icount   = 1
+  , irarity  = [(1, 5)]
+  , iverbHit = "thud"
+  , iweight  = 15000
+  , idamage  = 0
+  , iaspects = [ AddSkill SkMaxHP 14, AddSkill SkMaxCalm 80
+               , AddSkill SkSpeed 27, AddSkill SkNocto 3 -- good night vision
+               , AddSkill SkAlter (-2), SetFlag Durable ]  -- can't use stairs nor doors
+  , ieffects = []
+  , idesc    = "It has eight arms of rage."
+  , ikit     = [ ("tentacle 8", COrgan), ("small claw", COrgan)
```

Comment:
Well chosen organs, in particular sight --- 2 tiles better than human. As you can see here

https://github.com/AllureOfTheStars/Allure/blob/74ecdf787d3cbffca8a80c8ac6f5d20f187c941f/GameDefinition/Content/ItemKindOrgan.hs#L150

tentacles come in bunches of four. So it's enough to say `("tentacle", COrgan), ("tentacle", COrgan)` and we'll get eight of them.

OTOH, the claws come in twos --- is that right? I didn't read enough on octopuses, so I didn't verify if they have claws at the end of tentacles or something similar to mandibles in their mouths (beaks? we'd need to define such a new organ) or wherever else and in what number. They are mildly venomous, so we could add the `venomTooth`, if you so please.

Anyway, here starts the most fun part --- designing the special organ, or two, that makes octopus so different --- the siphon and the ink sack, if I got it right, or both in one. They could be similar to the vents but probably more interesting. We don't need to model all at once, especially if a large engine extension would be required, but we can certainly encode some ideas in the organ name(s), descriptions and comments. Let's brainstorm.

Reply:
I'm thinking the octopus should have 1) a beak, 2) suckers, and 3) ink sack. I will give a first attempt at an ink sack later.

Reply:
Go for it! 

Will the ink sac be the same as the siphon? No pressure to add both, it's fine to omit organs --- the ones we don't mention are considered to be a part of the trunk and mostly passive or to cooperate with the organs we do mention.